contrib/spring/src/main/java/io/opencensus/contrib/spring/autoconfig/OpenCensusProperties.java

```diff
@@ -35,4 +36,47 @@ public boolean isEnabled() {
   public void setEnabled(boolean enabled) {
     this.enabled = enabled;
   }
+
+  public Trace getTrace() {
+    return this.trace;
+  }
+
+  public void setTrace(Trace trace) {
+    this.trace = trace;
+  }
+
+  /** Trace properties. */
+  public static class Trace {
+
+    public static final String TRACE_PROPAGATION_TRACE_CONTEXT = "tracecontext";
```

Comment:
Nit: consider using Enum? Also public fields/methods need to have Javadoc.

Reply:
done.